library/archlinux

```diff
@@ -0,0 +1,12 @@
+# tarballs built by soyuz on a rootless environment
+# see https://github.com/archlinux/archlinux-docker/tree/no-root-build
+
+Maintainers: Santiago Torres-Arias <santiago@archlinux.org> (@SantiagoTores),
+             Christian Rebischke <Chris.Rebischke@archlinux.org> (@shibumi),
+             Pierre Schmitz <pierre@archlinux.de> (@pierres)
+GitRepo: https://github.com/archlinux/archlinux-docker.git
+
+
+Tags: latest, 20191006
+GitCommit: 671e950b6f879a346fffa2f9fbaaba8ace07efae
+GitFetch: refs/tags/20191006
```

Comment:
Gah, looks like https://github.com/archlinux/archlinux-docker/commits/20191006 has a tarball being modified in the history. :disappointed:

>  - [ ] if `FROM scratch`, tarballs only exist in a single commit within the associated history?

Reply:
ugh! I rebased the history there and removed that old tarball. Is this a reasonable fix or?

Reply:
Exactly perfect, LGTM! :tada: 

@yosifkit ? :innocent: :+1: